packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt

```diff
@@ -421,7 +421,7 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
         } else {
           mainActivity.let {
             Amplify.Auth.signInWithSocialWebUI(
-                    AuthProvider::class.java.getMethod(req.provider).invoke(null) as AuthProvider,
+                    AuthProvider::class.java.getMethod(req.provider!!).invoke(null) as AuthProvider,
```

Comment:
Huh! This field is [declared](https://github.com/aws-amplify/amplify-flutter/blob/c86f423fa1a72c805968bb395436bf9c9f10c379/packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/types/FlutterSignInWithWebUIRequest.kt#L21):
```kotlin
 val provider: String? = map?.get("authProvider") as String?;
```
What are the implications of `provider` being `null`, there? Will it ever reach this `!!`?

Reply:
No, the provider cannot be null! It's the else branch of `if(req.provider == null)`. Really weird that it's a compile issue while targeting newer sdk level(29).

Reply:
Oh whoops, I missed the `else`, sorry. :-D Proceed!